Core/Src/stm32f4xx_it.c

```diff
@@ -56,11 +57,12 @@
 /* USER CODE END 0 */
 
 /* External variables --------------------------------------------------------*/
+/*
 extern ETH_HandleTypeDef heth;
 extern CAN_HandleTypeDef hcan1;
 extern UART_HandleTypeDef huart3;
 extern TIM_HandleTypeDef htim1;
-
+*/
```

Comment:
If these are commented out, should they just be removed. Also, is it necessary for the hal can driver to be included in this file?